Sources/Commands/UserToolchain.swift

```diff
@@ -163,6 +146,30 @@ public struct UserToolchain: Toolchain {
         guard localFileSystem.isExecutableFile(clangCompiler) else {
             throw Error.invalidToolchain(problem: "could not find `clang` at expected path \(clangCompiler.asString)")
         }
+        return clangCompiler
+    }
+
+    public init(destination: Destination) throws {
+        self.destination = destination
+
+        // Get the search paths from PATH.
+        let searchPaths = getEnvSearchPaths(
+            pathString: getenv("PATH"), currentWorkingDirectory: localFileSystem.currentWorkingDirectory)
+
+        self.envSearchPaths = searchPaths
+
+        // Get the binDir from destination.
+        let binDir = destination.binDir
+
+        let swiftCompilers = try UserToolchain.determineSwiftCompilers(binDir: binDir, lookup: {UserToolchain.lookup(fromEnv: $0, searchPaths: searchPaths)})
```

Comment:
nit: needs a space after `{` and before `}`

Reply:
done